airflow/api_connexion/openapi/v1.yaml

```
@@ -1328,6 +1333,14 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/EventLog'
+    
+    Health:
```

```suggestion
    HealthInfo:
```
We have VersionInfo already. This will allow us to maintain consistency in names.